src/main/java/lotto/domain/Lottos.java

```diff
@@ -29,4 +30,11 @@ public LottoResult createLottoResult(WinningLotto winningLotto) {
 		}
 		return new LottoResult(lottoResult);
 	}
+
+	public Lottos addAll(Lottos autoLottos) {
+		List<Lotto> newLottos = new ArrayList<>();
+		newLottos.addAll(lottos);
+		newLottos.addAll(autoLottos.lottos);
+		return new Lottos(newLottos);
```

Comment:
스트림을 활용해볼 수도 있을 것 같아요 :)

```suggestion
        return Stream.concat(lottos.stream(), autoLottos.lottos.stream())
                .collect(Collectors.collectingAndThen(Collectors.toList(), Lottos::new));
```

Reply:
스트림 활용법 많이 배워갑니다 ㅠ